codecov.yml

```diff
@@ -1,15 +1,44 @@
+# See documentation at https://docs.codecov.io/docs/codecovyml-reference
+
+codecov:
+  notify:
+    # Wait for Jest & PHPUnit reports before notifying
+    after_n_builds: 2
+
 coverage:
   status:
-    patch: false
-    changes: false
+    # Patch-level coverage (how well is the PR tested)
+    patch:
+      default:
+        threshold: 5%
```

Comment:
This threshold is for the changes related to the PR (see https://docs.codecov.io/docs/commit-status#patch-status). I'm not sure if 5% is too much or too little - feedback on this would be appreciated.

Reply:
Isn't the current `0.1%` threshold we have OK to retain?

Reply:
The threshold for the patch status of the PR is a bit different, as it only measures lines adjusted in the PR. So if I were to add a function like so:

```php
function foo() {
    return 'bar';
}
```

The PR would be 0% covered, while the project coverage would slightly decrease. So if I add an accompanied test:

```php
function test_foo() {
    $this->assertEquals( 'bar', foo() );
}
```

The PR would then be 100% covered, while the project threshold will slightly increase.

Reply:
So if instead `foo` was defined as:

```php
function foo( $value = false ) {
    if ( ! $value ) {
        return 'bar';
    } else {
        return 'baz';
    }
}
```

Then the same test you had above:

```php
function test_foo() {
    $this->assertEquals( 'bar', foo() );
}
```

Would be a 50%?

Whereas if it were changed to:

```php
function test_foo() {
    $this->assertEquals( 'bar', foo() );
    $this->assertEquals( 'baz', foo( true ) );
}
```

Then it would be 100%?

Reply:
Yes, as I understand it.

Reply:
In that case, how about 75%?

Reply:
OK, and we can adjust that threshold as we go along.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/5341/commits/655d1418a155568d75de151928e3e5ab09948657.